### PR TITLE
add mruby package

### PR DESCRIPTION
**Pull request by dsisnero**

### CODE REVIEW

File: mingw-w64-mruby/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.4.0
+pkgrel=1
+pkgdesc="mruby is the lightweight implementation of the Ruby language"
+arch=('any')
+url="https://github.com/mruby"
+license=('MIT')
+makedepends=("${MINGW_PACKAGE_PREFIX}-ruby"
+             "bison"
+	    )
```
Comment (Alexpux) [inline mingw-w64-mruby/PKGBUILD]: Replace TABs everywhere with spaces. Format PKGBUILD

File: mingw-w64-mruby/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Maintainer: Dominic Sisners <dsisnero@gmail.com>
+
+_realname=mruby
+pkgbase=mingw-w64-${_realname}
+pkgbase=mingw-w64-${_realname}
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
+pkgver=1.4.0
+pkgrel=1
+pkgdesc="mruby is the lightweight implementation of the Ruby language"
```
Comment (Alexpux) [inline mingw-w64-mruby/PKGBUILD]: Add ````(mingw-w64)```` at the end of description

File: mingw-w64-mruby/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# Maintainer: Dominic Sisners <dsisnero@gmail.com>
+
+_realname=mruby
+pkgbase=mingw-w64-${_realname}
+pkgbase=mingw-w64-${_realname}
```
Comment (Alexpux) [inline mingw-w64-mruby/PKGBUILD]: Remove duplicate line

File: mingw-w64-mruby/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+             "bison"
+	    )
+
+source=(${_realname}-${pkgver}.zip::"https://github.com/${_realname}/${_realname}/archive/${pkgver}.zip")
+sha256sums=('449bf911a0cd3b92f39a0548341310f06626aa5f7c02f832853e86a06117bf39')
+
+
+build() {
+
+
```
Comment (Alexpux) [inline mingw-w64-mruby/PKGBUILD]: Remove empty lines

File: mingw-w64-mruby/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  cp build/host-debug/bin/* "${pkgdir}${MINGW_PREFIX}"/bin/ || return 1
+  cp build/host/bin/* "${pkgdir}${MINGW_PREFIX}"/bin/ || return 1
+
+  install -d "${pkgdir}${MINGW_PREFIX}"/lib
+  cp build/host/lib/*.a "${pkgdir}${MINGW_PREFIX}"/lib/
+
+  install -d "${pkgdir}${MINGW_PREFIX}"/include
+  cp -r include/* "${pkgdir}${MINGW_PREFIX}"/include/
+
+
```
Comment (Alexpux) [inline mingw-w64-mruby/PKGBUILD]: Remove empty lines

**Comment by dsisnero** [comment]
The failing test is not from the package- not sure why appveyor is failing.

**Pull request merged**